src/audio/tdfb/tdfb.c

```diff
@@ -459,6 +459,7 @@ static struct comp_dev *tdfb_new(const struct comp_driver *drv,
 	for (i = 0; i < PLATFORM_MAX_CHANNELS; i++)
 		fir_reset(&cd->fir[i]);
 
+	//cd->direction_updates = 1; /* Temporary force on */
```

Comment:
?

Reply:
Still no solution for this. I need to implement to testbench the controls set needed. Some script support would be ideal but it's tricky.